### PR TITLE
added homework regression

### PR DESCRIPTION
**Pull request by ThinamXx**
I have added a markdown for regression homework.

### CODE REVIEW

File: course-zoomcamp/cohorts/2022/02-regression/homework.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+For the rest of the homework, you'll need to use only these columns:
+
+* `'latitude'`,
+* `'longitude'`,
+* `'housing_median_age'`,
+* `'total_rooms'`,
+* `'total_bedrooms'`,
+* `'population'`,
+* `'households'`,
+*  'median_income',
```
Comment (alexeygrigorev) [inline course-zoomcamp/cohorts/2022/02-regression/homework.md]: probably also should put it in ``?

File: course-zoomcamp/cohorts/2022/02-regression/homework.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+* `'median_house_value'`
+
+Select only them.
+
+### Question 1
+
+Find a feature with missing values. How many missing values does it have?
+
```
Comment (alexeygrigorev) [inline course-zoomcamp/cohorts/2022/02-regression/homework.md]: Can you please add 4 options for this one and the rest too?

File: course-zoomcamp/cohorts/2022/02-regression/homework.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+## Submit the results
+
+Submit your results here: https://forms.gle/WCVb4KMNsgbkuJtr6
+
+If your answer doesn't match options exactly, select the closest one.
+
+## Deadline
+
+
+The deadline for submitting is , 17:00 CET. After that, the form will be closed.
```
Comment (alexeygrigorev) [inline course-zoomcamp/cohorts/2022/02-regression/homework.md]: let's do it on 19 september 23 cet

File: course-zoomcamp/cohorts/2022/02-regression/homework.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+## Navigation
+
+* [Machine Learning Zoomcamp course](../)
+* [Session 2: Machine Learning for Regression](./)
+* Previous: [Explore more](17-explore-more.md)
```
Comment (alexeygrigorev) [inline course-zoomcamp/cohorts/2022/02-regression/homework.md]: Not needed anymore

**Comment by alexeygrigorev** [comment]
Looks really good! just a few changes and it's ready

**Comment by ThinamXx** [comment]
I have updated the changes mentioned above.

**Comment by alexeygrigorev** [comment]
Thank you!

**Pull request merged**